pibooth/booth.py

```diff
@@ -328,13 +325,8 @@ def main_loop(self):
 
         finally:
             self.makers_pool.quit()
-            self.led_startup.quit()
-            self.led_preview.quit()
-            self.led_capture.quit()
-            self.led_print.quit()
+            self._plugin_manager.hook.pibooth_cleanup(app=self)
```

Comment:
In the hookspec, you have defined the config and window can be used.
Thus you have to pass kwargs: cfg=self._config and win=self._window

Reply:
I removed them from the hook spec for now.